docker/php/.env

```
@@ -0,0 +1,14 @@
+API_KEY=AD000000000000000000000000000000
+BASE_URL=http://php-api:80/
+MINIO_URL=http://minio:9000/
+DB_HOST_OPENML=database:3306
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_HOST_OPENML key should go before the MINIO_URL key

(UnorderedKey)